### DISCUSSION TITLE
Album Artist (TPE2) Required for Plex Agent Matching

### DISCUSSION THREAD

**Discussion by eriknh** [seanap/Plex-Audiobook-Guide#15] [General]
Hey there! 

I noticed in adding an m4b recently that if Album Artist wasn't applied to the M4B before moving over to the Plex media folder, then the agent didn't do the matching appropriately. I had all other tags there save for the Album Artist but matching was still listed as "Unknown Artist" in the Plex interface.

Be well!

**Comment by seanap** [comment]
Yes, plex uses the AlbumArtist as the Author.  Plex uses the TPE1 (ARTIST) tag as "All Artists on this track", this is typically "Author, Narrator" but I also use for Translator's, Editors, Forward, Co-authors, etc.

I will try to add a couple more columns to the Tag list to show what tags Plex and the Audiobook Agent look at, hopefully it will make it more clear.

The AlbumArtist is automatically set when scraped with the Audible mp3tag script (assuming there is a match), and when the mp3tag action script creates the Author folder it uses the AlbumArtist tag.

**Reply by eriknh** [reply]
Yeah awesome! I think your plan is perfect...also would you like some MacOS workflow input?

**Reply by seanap** [reply]
Absolutely, I'm sure there's a lot of Mac users out there that would appreciate it!  There is a blog post by Dustin Davis where he uses Hazel for a lot of the automations on MacOS https://dustindavis.me/blog/audible-to-plex
